_config.yml

```diff
@@ -63,6 +63,8 @@ aux_links:
     - "https://spec.openapis.org/oas/latest.html"
   "Latest Arazzo Specification":
     - "https://spec.openapis.org/arazzo/latest.html"
+  "Latest Overlay Specification":
+    - "https://spec.openapis.org/overlay/latest.html"
```

Comment:
Yeah, Jekyll and Liquid are horrible and have very few layout options.  Maybe  change "Specification" to "Spec" and change "Search OpenAPI Initiative Registry" to just "Search". because it does search the whole spec site AFAICT.

Reply:
@notEthan I truncated "specification" to "spec".  It turns out Jekyll/Liquid put whatever the site's title is in the Search box and I have no idea how to turn it off or change it, and I'm way past my Jekyll/Liquid frustration level at this point as they are the absolute worst I've been forced to work with and fill me with rage every time I attempt any improvement.  So someone else can deal with that.

Reply:
What I need to do let me know please thank you